tests/nemo_text_processing/zh/data_text_normalization/test_cases_decimal.txt

```diff
@@ -5,26 +5,31 @@
 -5.5555~负五点五五五五
 1890.5555~一千八百九十点五五五五
 20.123~二十点一二三
--2930.1929~负二千九百三十点一九二九
+-2930.1929~负两千九百三十点一九二九
 0.5~零点五
 5.0~五点零
 10.567~十点五六七
 -123.123~负一百二十三点一二三
 3123.1231~三千一百二十三点一二三一
-123123123.12312334234~一亿二千三百一十二万三千一百二十三点一二三一二三三四二三四
+123123123.12312334234~一亿两千三百一十二万三千一百二十三点一二三一二三三四二三四
 83888123.2398412~八千三百八十八万八千一百二十三点二三九八四一二
 283818.28994万~二十八万三千八百一十八点二八九九四万
-28394919.2312亿~二千八百三十九万四千九百一十九点二三一二亿
+28394919.2312亿~两千八百三十九万四千九百一十九点二三一二亿
 23.23万~二十三点二三万
 1233.1亿~一千二百三十三点一亿
 -123.213万~负一百二十三点二一三万
 123.890万~一百二十三点八九零万
 -233123.9940亿~负二十三万三千一百二十三点九九四零亿
 283943.234123亿~二十八万三千九百四十三点二三四一二三亿
-2391.2318~二千三百九十一点二三一八

```

Comment:
please make sure to have a few sentences for each `test_*.txt`, where the target semiotic class is in a context

Reply:
I would check every file to see if I have a few sentences in everyone of them.